package.json

```diff
@@ -26,7 +26,7 @@
   "author": "Ben Zörb <benjamin.zoerb@jvm.de>",
   "license": "MIT",
   "engines": {
-    "node": ">=14"
+    "node": ">=16"
```

Comment:
In my opinion we should only change the node required node version if this package breaks with node 14. 
Otherwise we would need a major version bump for this change

Reply:
Okay, I reverted the last commit. 

But then the package-lock doesn't seem to be correct. Node 14 actually comes with NPM 6. In the package-lock, however, the lockfileVersion is set to 2. With Node 14 and NPM 6, it would be set to 1.

Not so important. I just noticed that.